include/cudaq/Optimizer/Transforms/ArgumentDataStore.h

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
```

Comment:
nit-pick: we often add C++ here for `.h` files. 
```suggestion
/****************************************************************-*- C++ -*-****
```

Reply:
Often = always. 😄